server/lib/websockets/testing/websockets.ts

```diff
@@ -76,6 +79,7 @@ export class InspectableNydusClient extends NydusClient {
   // NOTE(tec27): We add an explicit type because it makes calling the method have better
   // documentation (the arguments are named, vs the arg_0, arg_1 stuff from Jest)
   publish: (path: string, data: any) => void = jest.fn()
+  unsubscribeClient: (client: NydusClient, path: string) => boolean = jest.fn()
```

Comment:
I'd just call this `unsubscribe`, `unsubscribeClient` is weird when this is already a Client. I also don't get why this needs to take a `NydusClient`? That should always be `===` itself or this call makes no sense